test/conformance/ingress/util.go

```diff
@@ -627,7 +622,7 @@ func CreateIngress(t *testing.T, clients *test.Clients, spec v1alpha1.IngressSpe
 		},
 		Spec: spec,
 	}
-	test.CleanupOnInterrupt(func() { clients.NetworkingClient.Ingresses.Delete(ing.Name, &metav1.DeleteOptions{}) })
+	t.Cleanup(func() { clients.NetworkingClient.Ingresses.Delete(ing.Name, &metav1.DeleteOptions{}) })
```

Comment:
Do we want to use EnsureCleanUp here (https://github.com/knative/networking/blob/4b262113649d4621e23c7267cab998e75d3cba6a/test/cleanup.go#L64) to make sure resources are cleaned up either on interrupt or when tests complete?